android-components/components/feature/addons/src/main/res/layout/mozac_feature_addons_header_section_item.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal"
+    android:padding="16dp">
+
+    <androidx.appcompat.widget.AppCompatImageView
+        android:id="@+id/warning_icon"
+        android:layout_width="24dp"
+        android:layout_height="24dp"
+        android:scaleType="center"
+        app:srcCompat="@drawable/mozac_ic_warning_fill_24"
+        app:tint="?android:attr/textColorLink" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:paddingTop="1dp"
```

Comment:
it aligns the icon correctly with the title

Reply:
ah, I see. Because we're using DP for the icon and SP for the text a `1dp` padding won't guarantee centering. Instead we'll want to have the icon and text centered in a `LinearLayout` so that whichever is larger will wrap the view size and the other will center within. Here is an example image with increased system font size and the suggested code change:

![Screenshot 2023-10-09 at 1 33 23 PM](https://github.com/mozilla-mobile/firefox-android/assets/13938063/d8a5054e-a7a4-4bbf-ae07-8c69b69f809c)

```suggestion
        <?xml version="1.0" encoding="utf-8"?>
<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

<LinearLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:padding="16dp"
    android:orientation="vertical"
    tools:ignore="RtlSymmetry">

    <LinearLayout
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:gravity="center_vertical">

        <androidx.appcompat.widget.AppCompatImageView
            android:id="@+id/warning_icon"
            android:layout_width="24dp"
            android:layout_height="24dp"
            android:scaleType="center"
            app:srcCompat="@drawable/mozac_ic_warning_fill_24"
            app:tint="?android:attr/textColorLink"/>

        <TextView
            android:text="@string/mozac_feature_addons_manager_notification_title_text"
            android:textSize="16sp"
            android:textColor="?android:attr/textColorPrimary"
            android:textStyle="bold"
            android:paddingStart="12dp"
            android:paddingEnd="0dp"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"/>

    </LinearLayout>

        <TextView
            android:text="@string/mozac_feature_addons_manager_notification_content_text"
            android:textColor="?android:attr/textColorPrimary"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:paddingVertical="16dp"
            android:paddingStart="36dp"/>

        <Button
            android:id="@+id/restart_button"
            style="@style/Widget.AppCompat.Button.Borderless.Colored"
            android:text="@string/mozac_feature_addons_manager_notification_restart_button"
            android:textColor="?android:attr/textColorLink"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_gravity="end"
            android:paddingVertical="8dp"/>

</LinearLayout>
```

Reply:
applied, thanks